packages/service-clients/azure-client/src/AzureClient.ts

```diff
@@ -50,6 +50,14 @@ const getTenantId = (connectionProperties: AzureConnectionConfig): string => {
 
 const MAX_VERSION_COUNT = 5;
 
+/**
+ * Setting this config to true indicates that only v2 clients will be collaborating together,
+ * such that runtime compability with v1 clients is not required.
+ *
+ * Defaults to false
+ */
+const v2NoCompatKey = "Fluid.Compatibility.V2Only";
```

Comment:
@scottn12 - My idea here is that consumers would only have to set this single config setting to release FF from having to support v1/v2 runtime compat.  As opposed to us publishing a laundry list of specific configs to set.

Another approach would be to add publicly exported "config packs" for v1 compat v. v2-only, and let consumers pass those configs in themselves.  Should we loop in a PM here?

Reply:
Sounds reasonable! Let's definitely loop in the PMs